src/Models/IoTProject.ts

```diff
@@ -494,29 +518,24 @@ export class IoTProject {
       fs.mkdirSync(deviceDir);
     }
 
-    // initialize the storage for azure component settings
-    const azureConfigFileHandler =
-        new azureComponentConfigModule.AzureConfigFileHandler(
-            this.projectRootPath);
-    azureConfigFileHandler.createIfNotExists();
-
     workspace.folders.push({path: constants.deviceDefaultFolderName});
     let device: Component;
-    if (boardId === az3166DeviceModule.AZ3166Device.boardId) {
-      device = new az3166DeviceModule.AZ3166Device(
-          this.extensionContext, this.channel, deviceDir,
-          projectTemplateItem.sketch);
-    } else if (boardId === ioTButtonDeviceModule.IoTButtonDevice.boardId) {
-      device = new ioTButtonDeviceModule.IoTButtonDevice(
-          this.extensionContext, deviceDir, projectTemplateItem.sketch);
-    } else if (boardId === esp32DeviceModule.Esp32Device.boardId) {
-      device = new esp32DeviceModule.Esp32Device(
-          this.extensionContext, this.channel, deviceDir,
-          projectTemplateItem.sketch);
-    } else if (boardId === raspberryPiDeviceModule.RaspberryPiDevice.boardId) {
+    // if (boardId === az3166DeviceModule.AZ3166Device.boardId) {

```

Comment:
Since other devices except RPI have been broken in the creation progress, I temperately comment out the creation of these devices.
TODO: @dilin-MS